reference/lib/ReferenceAmountDeriver.sol

```diff
@@ -63,7 +63,6 @@ contract ReferenceAmountDeriver is AmountDerivationErrors {
                 (endAmount * elapsed) +
                 extraCeiling);
 
-            // Division is performed without zero check as it cannot be zero.
```

Comment:
Should just replace this with a plain old description of what's happening

Reply:
No, it's just a text from her yet